motivation: initial commit (#5)
AevaOnline/supply-chain-synthesis: documents/motivation.md

What's most important, though, is that what we have is not enough for the threat complexity and sophistication of the future. We are learning about new vectors from live attacks faster than we can theorize and inventory them, and the prevalence of open source software in organizations of all types and in technology solutions with high societal impact calls for an evolved approach.

There are four main elements of the current explosion of ideas and value creation surge:

1. Practitioners across domains are rapidly prototyping ideas, the vast majority packaged as open source software, and communicating this across open communities
2. Already existing solutions in established domains are being reevaluated with a different perspective and in some case adapted to work in broader scenarios
3. Private (VC) investment in startups and other ventures that are primarily specializing in software supply chain security remains timid, especially when compared to other startup specialization areas, but has started to materialize
4. Large software companies are funding industry-wide initiatives, organizations and programs specifically to tackle software supply chain security challenges

> As with the attention surge from the public, idea flourishing is a net positive for supply chain security, but coordination challenges abound. These coordination challenges include poor reusability across use cases and scenarios due to siloed design choices, aggressive pace of change that breaks business continuity or creates operational risk and hindering adoption and outcomes due to poor interoperability.

Reviewer: minor grammar nits:
```suggestion
As public attention surges, the flourishing of many new ideas is a net positive for supply chain security, but coordination challenges abound. These coordination challenges include poor reusability across use cases and scenarios due to siloed design choices, aggressive pace of change that breaks business continuity or creates operational risk, and hinderance of adoption and outcomes due to poor interoperability.
```
